StudenSchool_DO/DbModels/ICopy.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace DbModels;
+
+internal interface ICopy<T>
```

Comment:
ICopyable is better name for interface. 'able' postfix is the best practice in this place for interface.